internal/x/xtabwriter/tabwriter.go

```diff
@@ -0,0 +1,34 @@
+package xtabwriter
```

Comment:
This is a great addition.

Reply:
Just wait till we get a struct tag in there
```go
type User struct {
  ID string `human:"hide"`
}
```